### PR TITLE
Initial Neo4j Support

### PR DESCRIPTION
**Pull request by Jotschi**

### CODE REVIEW

**Comment by Jotschi** [comment]
buildnow

**Comment by Jotschi** [comment]
buildnow

**Comment by Jotschi** [comment]
buildnow

File: databases/neo4j/src/main/java/com/gentics/mesh/neo4j/tx/Neo4jTx.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+	
+	@Override
+	public void close() {
+		try {
+			if (isSuccess()) {
+				commit();
+			} else {
+				rollback();
+			}
+		} catch (OConcurrentModificationException e) {
```
Comment (philippguertler) [inline databases/neo4j/src/main/java/com/gentics/mesh/neo4j/tx/Neo4jTx.java]: Do you want to use neo4j or orientdb?

File: services/metrics-prometheus/pom.xml [hunk 2]
```diff
@@ -40,6 +46,12 @@
 			<groupId>io.prometheus</groupId>
 			<artifactId>client</artifactId>
 			<version>0.0.10</version>
+			<exclusions>
+				<exclusion>
+					<groupId>javassist</groupId>
+					<artifactId>javassist</artifactId>
+				</exclusion>
+			</exclusions>
```
Comment (philippguertler) [inline services/metrics-prometheus/pom.xml]: Use dependency management instead

File: madl/api/src/main/java/com/gentics/mesh/madl/frame/EdgeFrame.java [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
-public interface EdgeFrame {
+public interface EdgeFrame extends ElementFrame, com.syncleus.ferma.EdgeFrame {
 
 	/**
 	 * Return the label of the edge.
 	 * 
 	 * @return
 	 */
-	String label();
+	default String label() {
```
Comment (philippguertler) [inline madl/api/src/main/java/com/gentics/mesh/madl/frame/EdgeFrame.java]: Move to element interface

**Pull request merged**